spec/controllers/auctions_controller_spec.rb

```diff
@@ -9,6 +9,45 @@
       expect(auction).to be_a(AuctionViewModel)
       expect(auction.id).to eq(auction_record.id)
     end
+
+    context 'the list of auctions is sorted' do
+      it 'renders them descending by datetime' do
+
+        Timecop.freeze
+
+        date_start = Time.now - 3.days
```

Comment:
hmm it looks like the difference between `Time.current` and `Time.now` is that `current` has a timezone fallback. Not sure how that would help with millisecond differences like I am seeing here (only on travis, not locally):

```
expected: 2016-05-20 20:55:16.235439941 +0000
     got: 2016-05-20 20:55:16.235439000 +0000
```